website/releases/release-1.0.0-beta1.md

```diff
@@ -0,0 +1,121 @@
+sequence.
+:::
+
+## Highlights
+
+### Format changes
+
+[HUDI-6242](https://issues.apache.org/jira/browse/HUDI-6242) is the main epic covering all the format changes proposals. The following are the main changes in this release:
+
+#### Timeline
+
+- Now all commit metadata is serialized to avro. This allows us to add new fields in the future without breaking
+  compatibility and also maintain uniformity in metadata across all actions.
+- All completed commit metadata file name will also have completion time. All the actions in requested/inflight states
+  are stored in the active timeline as files named <begin_instant_time>.<action_type>.<requested|inflight>. Completed
+  actions are stored along with a time that denotes when the action was completed, in a file named <
+  begin_instant_time>_<completion_instant_time>.<action_type>. This allows us to implement file slicing for non-blocking
+  concurrecy control.
+- Completed actions, their plans and completion metadata are stored in a more
+  scalable [LSM tree](https://en.wikipedia.org/wiki/Log-structured_merge-tree) based timeline organized in an *
+  *_archived_** storage location under the .hoodie metadata path. It consists of Apache Parquet files with action
+  instant data and bookkeeping metadata files, in the following manner. Checkout [timeline](/docs/next/timeline) docs for more details.
+
+#### Log File Format
+
+- Now in addition to the fields in the log file header mentioned in the [spec](https://hudi.apache.org/tech-specs/#log-file-format),
+  we also store the record positions in the header. This allows us to do position-based merging (apart from key-based merging) and skip pages based on positions.
+- Log file name will now have the deltacommit instant time instead of base commit instant time.
+
+#### Multiple base file formats
+
+Now you can have multiple base files formats in a Hudi table. Even the same filegroup can have multiple base file
+formats. We need to set a table config `hoodie.table.multiple.base.file.formats.enable` to use this features. And
+whenever we need to change the format, then just specify the format in the `hoodie.base.file.format"` config. Currently,
+only Parquet, Orc and HFile formats are supported.
+
+### Concurrency Control
+
+A new concurrency control mode called `NON_BLOCKING_CONCURRENCY_CONTROL` is introduced in this release, where unlike
+OCC, multiple writers can operate on the table with non-blocking conflict resolution. The writers can write into the
+same file group with the conflicts resolved automatically by the query reader and the compactor. The new concurrency
+mode is currently available for preview in version 1.0.0-beta only. You can read more about it under
+section [Model C: Multi-writer](/docs/next/concurrency_control#model-c-multi-writer). A complete example with multiple 
+Flink streaming writers is available [here](/docs/next/flink-quick-start-guide#non-blocking-concurrency-control). You
```

Comment:
Link will need to change after #10087 is merged since the content is moved away from quickstart to another page.